package.json

```diff
@@ -3,40 +3,40 @@
   "private": true,
   "version": "0.15.0-alpha",
   "devDependencies": {
-    "babel": "^5.8.3",
+    "babel": "^5.8.29",
     "babel-eslint": "4.1.3",
     "benchmark": "^1.0.0",
-    "browserify": "^9.0.3",
+    "browserify": "^12.0.0",
     "bundle-collapser": "^1.1.1",
     "coffee-script": "^1.8.0",
-    "del": "^1.2.0",
-    "derequire": "^2.0.0",
+    "del": "^2.0.2",
+    "derequire": "^2.0.3",
     "envify": "^3.0.0",
     "eslint": "1.7.3",
     "eslint-plugin-react": "3.6.3",
     "eslint-plugin-react-internal": "file:eslint-rules",
     "fbjs": "^0.4.0",
-    "fbjs-scripts": "^0.2.0",
+    "fbjs-scripts": "^0.3.0",
     "grunt": "^0.4.5",
     "grunt-cli": "^0.1.13",
     "grunt-compare-size": "^0.4.0",
     "grunt-contrib-clean": "^0.6.0",
-    "grunt-contrib-compress": "^0.13.0",
+    "grunt-contrib-compress": "^0.14.0",
     "gulp": "^3.9.0",
-    "gulp-babel": "^5.1.0",
-    "gulp-flatten": "^0.1.0",
-    "gulp-util": "^3.0.5",
+    "gulp-babel": "^5.3.0",
+    "gulp-flatten": "^0.2.0",
+    "gulp-util": "^3.0.7",
     "gzip-js": "~0.3.2",
-    "jest-cli": "^0.5.7",
+    "jest-cli": "^0.6.1",
     "jstransform": "^11.0.0",
```

Comment:
Want to just delete this while you're in here. I think I missed that in a rebase a couple months ago…